publications-2022.bib

```diff
@@ -168,6 +168,21 @@ @Misc{2022:fehling.bangerth:algorithms
   copyright = {arXiv.org perpetual, non-exclusive license}
 }
 
+@Article{2022:freese:heterogeneous,
+  author  = {Philip Freese},
```

Comment:
```suggestion
  author  = {Freese, Philip},
```
?

Reply:
bibtex understands it both ways. I agree that this way it is clearer.